service/codegen/src/main/java/io/helidon/service/codegen/ServiceDescriptorCodegen.java

```diff
@@ -1922,6 +1936,7 @@ private void postConstructMethod(ClassModel.Builder classModel, DescribedService
                     .addAnnotation(Annotations.OVERRIDE)
                     .addParameter(instance -> instance.type(typeName)
                             .name("instance"))
+                    .addContentLine("LOGGER.log(System.Logger.Level.DEBUG, \"postConstruct\");")
```

Comment:
The `Level` should be added as a class, so we can sort out imports (example above)